builders/pallets-precompiles/pallets/.pages

```diff
@@ -1,4 +1,4 @@
-title: Pallets
```

Comment:
I saw this under the "Pallets" Section:

<img width="571" alt="image" src="https://github.com/moonbeam-foundation/moonbeam-docs/assets/64150856/19d71858-2e1e-4845-8386-e3f2b0e12db5">

Unsure if it is because of the `--dirty` build. Production site looks OK

Reply:
Yeah, it is due to `--dirty` build